src/pages/index.js

```diff
@@ -6,7 +6,7 @@
 
 'use strict';
 
-import ButtonLink from './components/ButtonLink';
+import ButtonLink from '../templates/components/ButtonLink';
```

Comment:
Should `<ButtonLink>` be moved to `src/components`?

Reply:
Yes